### PR TITLE
Form copy tweaks

### PR DESCRIPTION
**Pull request by oliverjam**
1. We missed a "January 2022" when updating the EOI form for May cohort
2. Clarify that we only have EOIs from Jan 2021, so people who submitted before that know to re-do it

### CODE REVIEW

**Comment by oliverjam** [comment]
@yvonne-liu just saw this never got merged—are you happy with the extra clarification?

**Comment by yvonne-liu** [comment]
Oh I didn't see this 😬 and it might have a merge conflict with what I made earlier lol #176 

Also since this was made we've gone back to naming it Jan 2022 

This should be merged, though: 
> If you previously submitted an expression of interest before January 2021 you must [submit a new one](/forms/express-interest/) with the updated form.

**Comment by oliverjam** [comment]
It's ok. I can extract that bit into a new pr

File: src/forms/express-interest/index.md [hunk 1]
```diff
@@ -154,4 +154,4 @@ fields:
     maxLength: 250
 ---
 
-Complete this form if you're interested in the January 2022 cohort of Founders and Coders. We'll check your eligibility and get back to you with details on how to start your coding journey.
+Complete this form if you're interested in the May 2022 cohort of Founders and Coders. We'll check your eligibility and get back to you with details on how to start your coding journey.
```
Comment (oliverjam) [inline src/forms/express-interest/index.md:157]:
```suggestion
Complete this form if you're interested in the January 2022 cohort of Founders and Coders. We'll check your eligibility and get back to you with details on how to start your coding journey.
```

**Pull request merged**